main.js

```diff
@@ -0,0 +1,129 @@
+// A user should be able to enter a city into the url, click "Search" and get weather data on the city they entered.
+// A user should be able to see the current temperature (in imperial units).
+// A user should be able to see the current conditions (whether it's cloudy, raining, etc).
+// When a user does another search, their first search should be replaced.
+
+let weatherList = [];
+
+let weatherForecast = [];
+
+let $weather = $('#weather-list');
+
+
+let renderWeather = function () {
+  console.log('5');
```

Comment:
Treat this PR as if it were code that would be merged into production - you should remove dev comments like these console logs